libs/fluxc-plugin/src/main/kotlin/org/wordpress/android/fluxc/persistence/entity/BookingEntity.kt

```
@@ -34,6 +35,7 @@ data class BookingEntity(
     val parentId: Long,
     val personCounts: List<Long>?,
     val localTimezone: String,
+    @ColumnInfo(defaultValue = "") val attendanceStatus: AttendanceStatus,
```

Is this, as a default okay? All new bookings start with `booked` so we could do the same 🤔

---

Hmm, would that mean that for bookings created prior to adding the `attendance_status` change to the API, we'd be setting it to `booked`? 
What is `wp-admin` doing for old bookings that didn't have `attendance_status` set? I think we might simply want to leave it empty or `null` and avoid displaying any label or text when we don't have any real value. 
If a default value is to be added for old bookings, then I think it should be the backend doing so not us. 
WDYT?

---

>Hmm, would that mean that for bookings created prior to adding the attendance_status change to the API, we'd be setting it to booked?


Correct, that's probably not good. 

>What is wp-admin doing for old bookings that didn't have attendance_status set?

Good question, I will try to find it.

---

> Hmm, would that mean that for bookings created prior to adding the attendance_status change to the API, we'd be setting it to booked?

Given there are no CIAB production users, I think this should be fine 🤔 

The plugin seems to be using `booked` by default even for existing bookings as can be seen in L31 of `class-wc-booking.php`.

But for this special case, I think not setting a default is safest, we refresh the bookings each time they are open, so it's better to avoid hardcoding a default that can be changed by the plugin anytime, and hiding the value while we fetch the new value should be fine.

---

> think we might simply want to leave it empty or null and avoid displaying any label or text when we don't have any real value.

I've done that for now - updated in https://github.com/woocommerce/woocommerce-android/pull/14778/commits/d4e83cc680a8f34b331cb963f84048547ee13aa3